tsconfig.json

```diff
@@ -100,9 +103,6 @@
     {
       "path": "./packages/cactus-test-tooling/tsconfig.json"
     },
-    {
-      "path": "./packages/cactus-test-verifier-client/tsconfig.json"
-    },
     {
```

Comment:
@barnapa The GUI changes are looking good, but may I ask why did you remove the `packages/cactus-test-verifier-client/tsconfig.json` file from the typescript compilation altogether?

Reply:
ok added